<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>800</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Sitka Text</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Game of Life</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="currentIterationImage">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>600</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(58, 51, 255);
background-color: rgb(19, 74, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="simulateButton">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>470</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(37, 102, 255);
font: 75 10pt &quot;Sitka Text&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Simulate</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="simulationProgessBar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>601</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 127);</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QTextEdit" name="iterationCountTextEdit">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>40</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;Sitka Text&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>5</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 9pt &quot;Sitka Text&quot;;</string>
    </property>
    <property name="text">
     <string>Number of iterations</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>80</y>
      <width>541</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>20</width>
      <height>711</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="initalizeFromImageButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>120</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(37, 102, 255);
font: 75 10pt &quot;Sitka Text&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Initialize from image</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>160</y>
      <width>541</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="fileDisplayLabel">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>129</y>
      <width>341</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 8pt &quot;Sitka Text&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="positionX">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>210</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="positionY">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>210</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="width">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>250</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="height">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>300</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_x">
    <property name="geometry">
     <rect>
      <x>736</x>
      <y>180</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 9pt &quot;Sitka Text&quot;;</string>
    </property>
    <property name="text">
     <string>Position X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_x_2">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>180</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 9pt &quot;Sitka Text&quot;;</string>
    </property>
    <property name="text">
     <string>Position Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>250</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 9pt &quot;Sitka Text&quot;;</string>
    </property>
    <property name="text">
     <string>Width</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>300</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 9pt &quot;Sitka Text&quot;;</string>
    </property>
    <property name="text">
     <string>Height</string>
    </property>
   </widget>
   <widget class="QLabel" name="subSegmentLabel">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>270</y>
      <width>211</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>430</y>
      <width>531</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="addSubSegmentButton">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>382</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(37, 102, 255);
font: 75 10pt &quot;Sitka Text&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Add sub segment</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
